pkg/csi/cinder/controllerserver.go

```diff
@@ -66,7 +66,8 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	volType := req.GetParameters()["type"]
 
 	var volAvailability string
-	if req.GetAccessibilityRequirements() != nil {
+	// only check volume topo when it's enabled
+	if cs.Driver.withTypology && req.GetAccessibilityRequirements() != nil {
```

Comment:
Typology?

Reply:
sorry, didn't get what's wrong here.. @zetaab

Reply:
Instead of withTypology it should be withTopology?

Reply:
my bad, I always thought 'typology = topology' :(, just checked my dictionary ...
I will update here